custom_modules/gateway/interaction.js

```diff
@@ -21,8 +21,19 @@ class InteractionGateway extends NetworkGateway {
     return data.json();
   }
 
-  getComments = () => {
+  addComment = async (itemID, _userName, _comment) => {
```

Comment:
- [ ] the `_varName` could be removed so that the reputation of following JS Best Practices is intact 💪. example variable` name `comment` could be used instead of `_comment`.

Reply:
Corrected 💯